src/types/mongoose/community.ts

```diff
@@ -39,6 +40,6 @@ export interface ICommunityMethods {
 
 interface ICommunityQueryHelpers {}
 
-export interface CommunityModel extends Model<ICommunity, ICommunityQueryHelpers, ICommunityMethods> {}
+export type CommunityModel = Model<ICommunity, ICommunityQueryHelpers, ICommunityMethods>
```

Comment:
Is there a reason why you changed this? The way it was before is how the official Mongoose documentation defines models, and all the other model types are defined in that way, making this the only one to use a type alias instead of an interface https://mongoosejs.com/docs/typescript/statics-and-methods.html

Reply:
I don't remember doing that, perhaps i hit a "automatic fix" shortcut on a warning, we can change it back if you want.

Reply:
If you could. Functionally type aliases and interfaces are mostly identical, but I'd like to keep this in line with both the official docs and the rest of the codebase

Reply:
Done (6c592fb)